lib/riak/client/beefcake_protobuffs_backend.rb

```diff
@@ -13,12 +13,13 @@ def self.configured?
           require 'riak/client/beefcake/messages'
           require 'riak/client/beefcake/message_overlay'
           require 'riak/client/beefcake/object_methods'
+          require 'riak/client/beefcake/bucket_properties_operator'
           require 'riak/client/beefcake/crdt_operator'
           require 'riak/client/beefcake/crdt_loader'
           require 'riak/client/beefcake/protocol'
           require 'riak/client/beefcake/socket'
           true
-        rescue LoadError, NameError
+        rescue LoadError, NameError => e
```

Comment:
I so the exception goes into a variable and I can find it in a debugger. I should probably roll that one back.